toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -486,13 +486,6 @@ MOZ_EXPORT void RecordReplayInterface_Initialize(int* aArgc, char*** aArgv) {
 
   gUploadingRecording = RecordReplayValue("UploadingRecording", !!*dispatchAddress);
 
-  if (IsRecording()) {
-    // Print out a string that is recognized by the automated test harness.
-    AutoPassThroughThreadEvents pt;
-    const char* url = getenv("RECORD_REPLAY_URL");
-    fprintf(stderr, "CreateRecording %s %s\n", GetRecordingId(), url ? url : "");
-  }
```

Comment:
I think if we just move this to
```
diff --git a/devtools/server/actors/replay/module.js b/devtools/server/actors/replay/module.js
index 7398c85e88da..1f5f1426e108 100644
--- a/devtools/server/actors/replay/module.js
+++ b/devtools/server/actors/replay/module.js
@@ -26,6 +26,10 @@ const { Debugger, RecordReplayControl, Services, InspectorUtils } = sandbox;
 const isRecordingOrReplaying = !!RecordReplayControl.onNewSource;
 
 if (isRecordingOrReplaying) {
+  const recordingId = RecordReplayControl.recordingId();
+  if (recordingId) {
+    dump(`CreateRecording ${recordingId}\n`);
+  }
   Services.cpmm.sendAsyncMessage("RecordingStarting");
 }
 ```
 we'll get the behavior you are looking for, since we don't load `module.js` until the first checkpoint.